### DISCUSSION TITLE
bitrate=4000 and reslolution=15 but mpv player shows native resolution is 720p

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#434] [General]
I set up a slingbox entry in config.ini so the bitrate is 4000 and resolution is 15.
I am able to watch the stream in mpv player, but when I click shift+i to view the resolution, it shows 720p.
I think a 500 is able to stream at 1080p.
Why is it streaming at 720p?

Thanks.

**Comment by GerryDazoo** [comment]
The 500 cannot stream at 1080P best is 1080I at 30Hz. Don't confuse what the box will accept for an input and what it can output. 
Resolution 15 is 1920x544. The box will not up convert a signal, I suspect you really are feeding it a 720P signal. But if you think you have a real 1080 signal resolution 16 is the correct value.

**Comment by f520k510** [comment]
Thanks.  I set the resolution to 16 and mpv player shows 1080p as native resolution now.

**Reply by sp222222** [reply]
as stated the 500 only does 1080i not P.
